scripts/LCD-007.py

```diff
@@ -0,0 +1,21 @@
+import requests, json
```

Comment:
Add shebang line before import statement `#!/usr/bin/python3`
This way voicecommand can simply invoke your script without having to call `python3`